src/components/Header.js

```diff
@@ -102,15 +102,15 @@ const Header = () => {
             <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', height: 70 }}>
               <div style={{ display: 'flex', alignItems: 'center' }} aria-label="State of Washington">
                 <div style={{ textAlign: 'middle' }}>
-                  <img style={{ alignSelf: "left", paddingRight: 15 }} alt={"Wa State Seal"} width="120px" src="/imgs/doh_logo_doh-black.png" />
+                  {i18n.dir() == 'rtl' ? <img style={{ alignSelf: "left", paddingLeft: 15 }} alt={"Wa State Seal"} width="120px" src="/imgs/doh_logo_doh-black.png" /> : <img style={{ alignSelf: "left", paddingRight: 15 }} alt={"Wa State Seal"} width="120px" src="/imgs/doh_logo_doh-black.png" /> }
```

Comment:
The CSS standard says all measurements need units, so "paddingLeft: 15" should be "paddingLeft: 15px"  so we don't rely on browser specific behavior or quirks mode for things to work properly

Reply:
Additional changes have now been made. Please re-review.